plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -2152,6 +2152,22 @@ public final class CoreMetrics {
     .setHidden(true)
     .create();
 
+  /**
+   * @since 10.9
+   *
+   * This metric shows how many issues have a flag Prioritized Rule.
+   */
+  public static final String PRIORITIZED_RULE_ISSUES_KEY = "prioritized_rule_issues";
+  public static final Metric<Integer> PRIORITIZED_RULE_ISSUES = new Metric.Builder(
```

Comment:
The plugin API is intended to serve as a contract between analyzers and our products.

Is this new metric supposed to be used by analyzers (read or write)? By the name, I would say no, so it would be better to change it to an internal metric.

Reply:
Thanks for the reply @henryju , we will try using internal metric